docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+services:
+  app:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+    command: >
+      sh -c "python manage.py wait_for_db &&
+            python manage.py makemigrations &&
```

Comment:
You shouldn't have this command here (python manage.py makemigrations). Your migrations should already be present by now.